opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -103,6 +103,12 @@ message Span {
   // field must be empty. The ID is an 8-byte array.
   bytes parent_span_id = 4;
 
+  // Flags from the W3C traceparent, if provided.
+  // There are two flags defined in the W3C Trace Context Level 2 spec:
+  // - sampled = 0x1
+  // - random = 0x2
+  uint32 trace_flags = 17;
```

Comment:
This needs to be optional or have some other indicator of whether it was present or not, since zero is a valid flag combination

Reply:
I would like to know in what sense zero is a meaningful combination of flags? I see no reason to make an optional zero, since the present-zero and absent-zero should have identical meanings.

Reply:
Zero would mean that the span is not sampled. Such a span will not be sent in default configurations when using the OTel SDK, but in principle is valid (e.g. you might want to debug some sampler that throws away too many spans).